db/migrate/20180801183223_create_taxon_scoring_models.rb

```diff
@@ -0,0 +1,10 @@
+class CreateTaxonScoringModels < ActiveRecord::Migration[5.1]
+  def change
+    create_table :taxon_scoring_models do |t|
```

Comment:
Would it make sense now to associate this with a user/owner? Would we ever expose this?
I kind of want to add that ability to all our content for accountability for security purposes..

Reply:
I can add that but probably won't fill it for a while.